.github/workflows/ci-credit-application-service.yml

```diff
@@ -31,10 +33,38 @@ jobs:
     - name: Build with Maven
       run: mvn -B verify --file domain-service/credit-application/pom.xml
 
-    #- name: Publish to GitHub Packages Apache Maven
-    #  run: mvn deploy --file domain-service/credit-application/pom.xml -DskipTests
-    #  env:
-    #    GITHUB_TOKEN: ${{ github.token }}
+#    - if: endsWith(github.REF, '/master') != true
```

Comment:
Do we need this commented lines?

Reply:
sorry, forgot to remove it.
will update.